src/test/java/org/jpeek/MetricsTest.java

```diff
@@ -150,7 +150,15 @@
             new Object[] {"MethodsWithDiffParamTypes", "PCC", 0.3333d},
             new Object[] {"Foo", "OCC", 0.5d},
             new Object[] {"Foo", "TCC", 1.0d},
-            new Object[] {"MethodsWithDiffParamTypes", "TCC", 0.2d}
+            new Object[] {"MethodsWithDiffParamTypes", "TCC", 0.2d},
```

Comment:
@llorllale I understand these tests work because these classes do not have transitive connections longer than 3?

Reply:
@amihaiemil yes